packages/web-twig/src/Resources/components/Toast/ToastBar.twig

```diff
@@ -53,7 +53,7 @@
                    <Icon boxSize="20" name="{{ _iconNameValue }}" isReusable={ false } />
                 {% endif %}
             {% endif %}
-            <div class="{{ _messageClassName }}" data-spirit-populate-field="message">
+            <div class="{{ _containerClassName }}" data-spirit-populate-field="message">
```

Comment:
the data-spirit-populate-field="message" should be on the message and not the container.